etc/init/spinnaker.conf

```diff
@@ -4,22 +4,36 @@ start on filesystem or runlevel [2345]
 stop on shutdown
 
 pre-start script
-    if [ -f /opt/spinnaker/cassandra/SPINNAKER_INSTALLED_CASSANDRA ]; then
-        if ! nc -z localhost 9042; then
-            echo "Waiting for Cassandra"
-            for i in `seq 1 30`; do
-                if nc -z localhost 9042; then
-                    echo "Cassandra is ready"
-                    break
-                fi
-                if [ $i -eq 30 ]; then
-                    echo "Cassandra is not up on localhost.
-                    Start it, and then restart Spinnaker."
-                fi
-                sleep 1
-            done
+  if [ -f /opt/spinnaker/cassandra/SPINNAKER_INSTALLED_CASSANDRA ]; then
+    if ! nc -z localhost 9042; then
+      echo "Waiting for Cassandra"
+      for i in `seq 1 30`; do
+        if nc -z localhost 9042; then
+          echo "Cassandra is ready"
+          break
         fi
+        if [ $i -eq 30 ]; then
+          echo "Cassandra is not up on localhost.
+          Start it, and then restart Spinnaker."
+        fi
+        sleep 1
+      done
     fi
+  fi
+
+  if ! nc -z localhost 6379; then
```

Comment:
Once someone takes their redis instance and moves it to another node they will no longer be able to run spinnaker